src/interpolation/compute/pipe.test.js

```diff
@@ -0,0 +1,72 @@
+import pipe from './pipe'
+import { UnknownFormatterError, UnvalidFormatterError } from './errors'
+
+test('One transformation - without parenthesis', () => {
+  const value = pipe(
+    'Thibaud',
+    'lowercase',
+    {}
+  )
+
+  expect(value).toBe('thibaud')
+})
+
+test('One transformation - with parenthesis, no args', () => {
+  const value = pipe(
+    'Thibaud',
+    'lowercase()',
+    {}
+  )
+
+  expect(value).toBe('thibaud')
+})
+
+test('One transformation - multiple args', () => {
+  const value = pipe(
+    '1500',
+    'money(2, "$")',
+    {}
+  )
+
+  expect(value).toBe('1500.00$')
+})
+
+test('One transformation - with variable args', () => {
+  const value = pipe(
+    '1500',
+    'money(2, dollard)',
```

Comment:
I think the variable name should be "dollar" instead of "[dollard](https://fr.wikipedia.org/wiki/Dollard)" ^^